### PR TITLE
Illustrate concepts on home

### PR DESCRIPTION
**Pull request by Ndpnt**
screencapture-localhost-4000-encyclopedia-2019-06-07-18_12_32

### CODE REVIEW

File: _pages/en/encyclopedia/index.md [hunk 1]
```diff
@@ -6,19 +6,81 @@ nav_order: 1
...
+While useful to inform the best course of action, this phase is not needed to react. Besides, attribution does not share the urgency that an impactful reaction depends on. Related [practices](attribution/case-studies) and [tools](attribution/tools) are part of forensics and open-source intelligence (OSINT). Attribution brings the most value by linking together seemingly independent campaigns, thus allowing to refine detection and reaction means throught the unveiling of patterns.
+
+### Research
+#### Going beyond specific campaigns, [research](research) adds to the knowledge of information manipulation.
+{: .thin}
+
+![](/assets/img/home/research.png){:width="50%" .floatright .ml-4 .pt-4 .shape-research } [Studying](research/practices) past disinformation efforts allows to more easily identify and assess future ones. It can also be used to [evaluate](research/tools) the effectiveness of tools and practices that counter them. Information manipulation has no dedicated academic field, most of the [current research](research/actors) on the topic happens at the intersection of social and computer sciences.
```
Comment (MattiSG) [inline _pages/en/encyclopedia/index.md]: Shouldn't the image and the text be on their own lines?

File: _pages/en/encyclopedia/index.md [hunk 2]
```diff
@@ -6,19 +6,81 @@ nav_order: 1
...
+![](/assets/img/home/prevention.png){:width="46%" .floatright .ml-4 .py-2 .shape-prevention } Disinformation targets human societies. While research can help us understand how attacks work, and the detection-qualification-reaction sequence can defuse many of them, the most viable long-term solution is to immunize the entire social body.
+{: .fix-height-with-floated-element }
+
+- - -
+
+### Foster collaboration and increase impact
+#### This collaborative resource aims at empowering all actors countering information manipulation to grow and improve.
+{: .thin}
+
+To do so, it unifies vocabulary, for instance by providing concepts like the ones you just read about. It also gathers exemplary, sharable case studies and describes the best **[practices](/encyclopedia/detection/practices)**, **[tools](/encyclopedia/detection/tools)** and **[actors](/encyclopedia/detection/actors)** in the field. Furthermore, it consolidates opportunities such as [funding](/encyclopedia/funders) and [events](/encyclopedia/agenda).
```
Comment (MattiSG) [inline _pages/en/encyclopedia/index.md]:
```suggestion
To do so, it unifies vocabulary and provides concepts like the ones you just read. It also gathers exemplary, sharable case studies and describes the best **[practices](/encyclopedia/detection/practices)**, **[tools](/encyclopedia/detection/tools)** and **[actors](/encyclopedia/detection/actors)** in the field. Furthermore, it consolidates opportunities such as [funding](/encyclopedia/funders) and [events](/encyclopedia/agenda).
```

File: _sass/custom/custom.scss [hunk 3]
```diff
@@ -148,3 +148,68 @@ iframe {
...
+.shape-outside-ellipse {
+  shape-outside: ellipse();
+}
+
+.shape-detection {
+  shape-outside: polygon(7% 1%, 49% 0, 100% 18%, 100% 39%, 44% 100%, 33% 100%, 0 82%, 1% 38%);
+}
+
+.shape-qualification {
+  shape-outside: polygon(0 63%, 0 0, 100% 0, 100% 100%, 31% 100%, 31% 89%, 25% 83%, 22% 77%, 20% 68%, 20% 63%);
```
Comment (MattiSG) [inline _sass/custom/custom.scss]: `shape-outside` is [not supported](https://caniuse.com/#search=shape-outside) by Edge nor IE.
How does that degrade?

File: _pages/en/encyclopedia/index.md (hunk 1 above)
Comment (Ndpnt) [reply _pages/en/encyclopedia/index.md]: Nope, as it they are in the same <p> tag

File: _sass/custom/custom.scss (hunk 3 above)
Comment (Ndpnt) [reply _sass/custom/custom.scss]: Like this:
screencapture-localhost-4000-encyclopedia-2019-06-07-19_03_55

**Pull request closed without merging**